.github/workflows/postsubmit.yml

```diff
@@ -40,10 +40,14 @@ jobs:
       # analyze Dart for errors
       - name: "Analyze code"
         run: flutter analyze --fatal-infos --fatal-warnings .
-      
+
       # Run tests
       - name: "Run Tests"
         run: ci/testing.sh
+        
+      # Code Coverage
+      - name: "Run Tests" 
+        run: flutter test --coverage
```

Comment:
Is there any effect from using it, as far as I understood it it just generates a html file

Reply:
It's a draft at this point, eventually, it should do this: https://github.com/openfoodfacts/robotoff/pull/665

Reply:
Shouldn't it go to the presubmit workflow?

Reply:
Sure